account_clearance_plan/models/account_invoice.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = "account.invoice"
+
+    def get_clearance_plan_wizard(self):
+        move_lines = self.env["account.move.line"]
+        for rec in self:
+            move_lines |= rec.move_id.line_ids.filtered(
+                lambda l: l.account_id == rec.account_id
+            )
+        context = {"move_line_ids": move_lines.ids}
```

Comment:
```suggestion
        context = {"active_ids": move_lines.ids, "active_model": "accout.move.line"}
```

Would this work and simplify the `default_get` in the wizard?